demangle-mode.el

```diff
@@ -128,13 +143,15 @@ transaction queue restarts automatically when needed."
 	   (if (fboundp 'make-process)
 	       ;; Emacs 25 and later
 	       (make-process :name "demangler"
-			     :command '("c++filt" "--no-strip-underscore")
+			     :command (demangle--cxxfilt-command)
 			     :noquery t
 			     :connection-type 'pipe
 			     :sentinel #'demangle--sentinel)
 	     ;; Emacs 24.x and earlier
 	     (let* ((process-connection-type nil)
-		    (subprocess (start-process "demangler" nil "c++filt" "--no-strip-underscore")))
+		    (subprocess (start-process "demangler" nil
+                                               ,demangle-cxxfilt-executable
```

Comment:
This cannot be right: `,` and `,@` are only meaningful inside a backtick (`` ` ``) construct. Let’s correct that and also promote code reuse by using the new `demangle--c++filt-command` function. Please change this `start-process` call to:

``` elisp
(apply #'start-process "demangler" nil (demangle--c++filt-command))
```